SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -111,6 +111,8 @@ namespace SpatialConstants
 	const Worker_ComponentId CLIENT_RPC_ENDPOINT_COMPONENT_ID				= 9990;
 	const Worker_ComponentId SERVER_RPC_ENDPOINT_COMPONENT_ID				= 9989;
 	const Worker_ComponentId NETMULTICAST_RPCS_COMPONENT_ID					= 9987;
+	const Worker_ComponentId NOT_SPAWNED_COMPONENT_ID						= 9986;
```

Comment:
Not sure about this name. `ACTOR_WAS_LOADED_COMPONENT_ID` ? `LEVEL_ACTOR_COMPONENT_ID` ?

Reply:
It's bad name, should be changed.